docs/useGetList.md

```diff
@@ -118,3 +118,40 @@ const LatestNews = () => {
 ```
 
 The `data` will automatically update when a new record is created, or an existing record is updated or deleted.
+
+## TypeScript
+
+The `useGetList` hook accepts a generic parameter for the record type:
```

Comment:
You should explain the benefit that this brings in text before the snippet.

Reply:
Explain the benefit of typescript?